fair_mango/metrics/metrics.py

```diff
@@ -24,7 +24,27 @@
 
 
 class SelectionRate(Metric):
-    """Calculate selection rate for different sensitive groups"""
+    """Calculate selection rate for different sensitive groups.
```

Comment:
I'm not a fan of `for different sensitive groups` as it does not seem to reflect what this rellay does. But i have nothing else to suggest